tests/scripts/build_minimal_appimage_with_custom_apprun.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+if [[ -z ${ARCH+x} ]]; then
+    echo "ARCH not set."
+    exit -1;
+fi
+
+LINUXDEPLOY=$1
+if [[ -z ${LINUXDEPLOY+x} ]]; then
+    echo "LINUXDEPLOY path not set."
```

Comment:
This entire parsing gives a wrong impression of the use of this script. You should rather show a usage method if the number of arguments is too low.

Reply:
Was dropped